velox/expression/tests/ExpressionFuzzer.cpp

```diff
@@ -405,6 +408,17 @@ class ExpressionFuzzer {
     }
   }
 
+  void appendConjunctSignatures() {
+    CallableSignature conjunctAndSignature;
+    conjunctAndSignature.name = "and";
+    conjunctAndSignature.returnType = BOOLEAN();
+    conjunctAndSignature.args = {BOOLEAN(), BOOLEAN()};
+    signatures_.emplace_back(conjunctAndSignature);
+
+    conjunctAndSignature.name = "or";
```

Comment:
nit: Maybe rename it as "conjunctSignature" since we assign "or" to it.

Reply:
Yeah, will change